src/addrman.cpp

```diff
@@ -1,6 +1,6 @@
-// Copyright (c) 2012 Pieter Wuille
-// Distributed under the MIT software license, see the accompanying
-// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+/// Copyright (c) 2012 Pieter Wuille
+/// Distributed under the MIT software license, see the accompanying
+/// file COPYING or http://www.opensource.org/licenses/mit-license.php.
```

Comment:
We don't want any of these comments included in Doxygen, as they don't add value to the dev documentation.